### PR TITLE
MEPTS-1014: Using the right concepts for ficha in desaggregation for …

### PR DESCRIPTION
**Pull request by psmatsinhe**
…more than 6 months

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java [hunk 1]
```diff
@@ -1618,7 +1618,7 @@ public SqlCohortDefinition getPatientsWithSemiAnnualTypeOfDispensation() {
             + "                 INNER JOIN obs o ON e.encounter_id = o.encounter_id"
             + "                 WHERE"
             + "                     e.encounter_type = ${6}"
-            + "                         AND (o.value_coded IN (${1256} , ${1257}) OR o.concept_id = ${23739})"
+            + "                         AND (o.concept_id= 23730 OR o.concept_id = 23739 OR o.concept_id = 23888)"
```
Comment (helderjosue) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java]: @psmatsinhe 
These Concepts should not be indicated directly, please add **${...}**

**Review by helderjosue — APPROVED** [review]
After applying the changes indicated on the comment it will be good to go.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java (hunk 1 above)
Comment (ningosi) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java]: The change was this easy?  I see now the mars journey story from @psmatsinhe

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java [hunk 2]
```diff
@@ -1618,7 +1618,7 @@ public SqlCohortDefinition getPatientsWithSemiAnnualTypeOfDispensation() {
             + "                 INNER JOIN obs o ON e.encounter_id = o.encounter_id"
             + "                 WHERE"
             + "                     e.encounter_type = ${6}"
-            + "                         AND (o.concept_id= 23730 OR o.concept_id = 23739 OR o.concept_id = 23888)"
+            + "                         AND (o.concept_id= ${23730} OR o.concept_id = ${23739} OR o.concept_id = ${23888})"
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java]: what if you would have done AND concept_id IN(${23730}, ${23739},${23888}) instead?

**Pull request merged**